### PR TITLE
Support default values for options

### PR DESCRIPTION
**Pull request by jbrumwell**
This should resolve the default value portion of this ticket, https://github.com/dthree/vorpal/issues/52
- Convert command.js to es6
- Change signature of option method to `option(flags, description, options)` backwards compatibility with previous signature
- Fixed linting errors in a couple of files
- Added an `options.default` property that allows for setting default values
- Added two tests for default values

### CODE REVIEW

File: lib/option.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-'use strict';
```
Comment (dthree) [inline lib/option.js]: Let's keep the `'use strict'` for now, for consistency.

File: lib/option.js [hunk 2]
```diff
@@ -12,12 +12,17 @@ class Option {
    * @api public
    */
 
-  constructor(flags, description, autocomplete) {
+  constructor(flags, description, options = {}) {
+    options = _.isPlainObject(options) ? options : {
```
Comment (dthree) [inline lib/option.js]: I'm trying to get lodash out of the project eventually - is it possible to use vanilla JS for this?

File: lib/option.js [hunk 3]
```diff
@@ -12,12 +12,17 @@ class Option {
...
+      autocomplete: options
+    };
+
+    this.autocomplete = _.get(options, 'autocomplete');
```
Comment (dthree) [inline lib/option.js]: This as well.

**Comment by dthree** [comment]
Thanks. I like the idea, as well as your handling of autocomplete, which is taking the third parameter. Please fix the conflicts and the points I brought up and we should be good to go!

**Comment by milesj** [comment]
Was just about to file an issue about this and saw the PR. 👏

**Comment by jbrumwell** [comment]
@dthree changes made and conflicts avoided :)

**Comment by milesj** [comment]
@jbrumwell Can you rebase off master and make sure everything still works? I added Yarn support and updated all deps.

**Comment by milesj** [comment]
I'll go ahead and merge this and fix any inconsistencies, seeing as how the 2.0 version is a ways out. Thanks for the work!

**Pull request merged**